src/crimson/os/seastore/segment_cleaner.cc

```diff
@@ -86,6 +86,7 @@ int64_t SpaceTrackerDetailed::SegmentMap::allocate(
     }
     bitmap[i] = true;
   }
+  total_occupied += len;
```

Comment:
IIUC `calc_utilization()` is calculated by alive/allocated for each segment, why it is better than alive/segment-size? Seems to me the segment utilization is more accurate to be the latter one.